stdlib/fun.mli

```diff
@@ -61,3 +63,67 @@ exception Finally_raised of exn
     an unexpected exception or a programming error. As a general rule,
     one should not catch a [Finally_raised] exception except as part of
     a catch-all handler. *)
+
+(** {1:examples Examples}
+
+    {2 Combinators}
+
+    One common mode of usage for {!combinators} is to create and pass anonymous
+    functions as arguments to higher-order functions, without needing to specify
+    [(fun ... -> ...)] explicitly.
```

Comment:
Personally, I never use combinators to avoid writing an explicit lambda. Factorizing common patterns is a much better motivation from my point of view (which tends to exclude toy combinators like  `flip` and `curry` or `uncurry`).

Reply:
> Factorizing common patterns
could you please expand on this a bit more? I can add it to the passage or just be more neutral and snip this part

Reply:
Maybe the focus on anonymous function is not necessary either:

> One common mode of usage for {!combinators} is to create short-lived arguments for higher-order functions

Or maybe we should emphasis the lightweight idea rather constrasting with abstraction:

>  {{!combinators}Combinators} provide a lightweight way to create short-lived arguments for higher-order functions